pytorch/inference/docker/1.12/py3/sdk2.5.0/Dockerfile.neuron

```diff
@@ -0,0 +1,152 @@
+FROM ubuntu:20.04
+
+#SDK 1.17.1 has version 1. We skipped 1.18.0.
+LABEL dlc_major_version="2"
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG PYTHON=python3.8
+ARG PYTHON_VERSION=3.8.13
+ARG TS_VERSION=0.5.3
+ARG MAMBA_VERSION=4.12.0-2
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+ENV LD_LIBRARY_PATH /lib/x86_64-linux-gnu:/opt/conda/lib/:$LD_LIBRARY_PATH
+ENV PATH /opt/conda/bin:/opt/aws/neuron/bin:$PATH
+ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
+ENV TEMP=/home/model-server/tmp
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends software-properties-common \
```

Comment:
Recommend adding the apt-get upgrade to pick up updated "base" packages, catching OS CVE fixes.